autoware_launch/rviz/autoware.rviz

```diff
@@ -1332,6 +1332,18 @@ Visualization Manager:
                             Reliability Policy: Reliable
                             Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/wall_marker
                           Value: true
+                        - Class: rviz_default_plugins/MarkerArray
+                          Enabled: true
+                          Name: VirtualWall (ObstacleCruise)
+                          Namespaces:
+                            {}
+                          Topic:
+                            Depth: 5
+                            Durability Policy: Volatile
+                            History Policy: Keep Last
+                            Reliability Policy: Reliable
+                            Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_cruise_planner/debug/stop_wall_marker
```

Comment:
![image](https://user-images.githubusercontent.com/8327598/172110366-935e608a-db7c-4723-893f-846789553f70.png)
 ```suggestion
                            Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_cruise_planner/virtual_wall
```

Reply:
You mean, you want me to change the topic name?

Reply:
@saito

Reply:
Sorry, I'm editing the comment.
Can you please change the name to match the other virtual wall tpoic?

Reply:
Got it